src/main/java/com/github/curriculeon/SimplePhoneBook.java

```diff
@@ -1,32 +1,46 @@
 package com.github.curriculeon;
 
-import java.util.List;
+import java.util.*;
 
 /**
  * Created by leon on 12/16/2019.
  */
 public class SimplePhoneBook {
-    public void add(String name, String phoneNumberToAdd) {

```

Comment:
I would recommend creating a field of type `AdvancedPhoneBook` in this class.
Then, use defer all method calls to `advancedPhoneBook` instance.